src/test/scripts/elasticsearch-values.yaml

```diff
@@ -0,0 +1,21 @@
+---
+
+master:
+  replicas: 1
+  podAnnotations:
+    "atlassian.com/business_unit": "server_engineering"
```

Comment:
Is this necessary in our cluster?

Reply:
Only for running in KITT. It's probably not necessary in day-to-day, or for CI, but there's no harm in adding it for future use.